### DISCUSSION TITLE
Dynamically add errors when using useFieldArray

### DISCUSSION THREAD

**Discussion by MikeBish13** [react-hook-form/react-hook-form#4897] [Q&A]
How would you go about dynamically adding custom error messages to inputs that are generated with useFieldArray?

Very simple codesandbox here - https://codesandbox.io/s/romantic-dubinsky-xd6f7?file=/src/App.js

Usually I would write something like `{errors.name && <p>This can't be empty</p>` however with useFieldArray the names of the inputs are dynamically generated. I've tried something like `{errors.items.index.name && <p>This can't be empty</p>` but that doesn't work. 

Any suggestions?

**Comment by bluebill1049** [comment]
`errors.items[index]` should contain your error object.
